compiler/testData/diagnostics/testsWithStdLib/coroutines/restrictSuspension/outerYield_1_3.fir.kt

```diff
@@ -87,7 +87,7 @@ fun test() {
                 this@with.<!UNRESOLVED_REFERENCE!>yield<!>("")
 
                 yield2("")
-                this@with.<!INAPPLICABLE_CANDIDATE!>yield2<!>("")
+                this@with.<!UNSAFE_CALL!>yield2<!>("")
```

Comment:
Similar to `kt557`, this should be `UNRESOLVED_REFERENCE_WRONG_RECEIVER`: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/testsWithStdLib/coroutines/restrictSuspension/outerYield_1_3.kt#L90